mail_composer_cc_bcc_account/wizards/account_move_send.py

```diff
@@ -0,0 +1,139 @@
+                )
+                wizard.partner_bcc_ids = self._get_default_mail_partner_bcc_ids(
+                    self.move_ids, wizard.mail_template_id
+                )
+            else:
+                wizard.partner_cc_ids = None
+                wizard.partner_bcc_ids = None
+
+    def _get_mail_move_values(self, move, wizard=None):
+        mail_template_id = (
+            move.send_and_print_values
+            and move.send_and_print_values.get("mail_template_id")
+        )
+        mail_template = (
+            wizard
+            and wizard.mail_template_id
+            or self.env["mail.template"].browse(mail_template_id)
+        )
+        mail_lang = self._get_default_mail_lang(move, mail_template)
+        return {
+            "mail_template_id": mail_template,
+            "mail_lang": mail_lang,
+            "mail_body": wizard
+            and wizard.mail_body
+            or self._get_default_mail_body(move, mail_template, mail_lang),
+            "mail_subject": wizard
+            and wizard.mail_subject
+            or self._get_default_mail_subject(move, mail_template, mail_lang),
+            "mail_partner_ids": wizard
+            and wizard.mail_partner_ids
+            or self._get_default_mail_partner_ids(move, mail_template, mail_lang),
+            "mail_attachments_widget": wizard
+            and wizard.mail_attachments_widget
+            or self._get_default_mail_attachments_widget(move, mail_template),
+            "partner_cc_ids": wizard
+            and wizard.partner_cc_ids
+            or self._get_default_mail_partner_cc_ids(move, mail_template),
+            "partner_bcc_ids": wizard
+            and wizard.partner_bcc_ids
+            or self._get_default_mail_partner_bcc_ids(move, mail_template),
+        }
+
+    # -------------------------------------------------------------------------
+    # BUSINESS ACTIONS
+    # -------------------------------------------------------------------------
+
+    @api.model
+    def _send_mail(self, move, mail_template, **kwargs):
+        """Send the journal entry passed as parameter by mail."""
+        partner_ids = kwargs.get("partner_ids", [])
+
+        new_message = move.with_context(
+            no_new_invoice=True,
+            mail_notify_author=self.env.user.partner_id.id in partner_ids,
+            is_from_composer=True,
+            partner_cc_ids=self.partner_cc_ids,
+            partner_bcc_ids=self.partner_bcc_ids,
+        ).message_post(
+            message_type="comment",
+            **kwargs,
+            **{
+                "email_layout_xmlid": "mail.mail_notification_layout_with_responsible_signature",  # noqa: E501
+                "email_add_signature": not mail_template,
+                "mail_auto_delete": mail_template.auto_delete,
+                "mail_server_id": mail_template.mail_server_id.id,
+                "reply_to_force_new": False,
+            },
+        )
```

Comment:
not blocking, but I would prefer having this in more steps.
```suggestion
        move_with_context = move.with_context(
            no_new_invoice=True,
            mail_notify_author=self.env.user.partner_id.id in partner_ids,
            is_from_composer=True,
            partner_cc_ids=self.partner_cc_ids,
            partner_bcc_ids=self.partner_bcc_ids,
        )
        extra_args = {
            "email_layout_xmlid": "mail.mail_notification_layout_with_responsible_signature",
            "email_add_signature": not mail_template,
            "mail_auto_delete": mail_template.auto_delete,
            "mail_server_id": mail_template.mail_server_id.id,
            "reply_to_force_new": False,
            "message_type": "comment" 
        }
        kwargs.update(extra_args)
        move_with_context.message_post(**kwargs)